### ISSUE TITLE
Graph title limit too small

### ISSUE THREAD

**Issue by jlaw9** [Murali-group/GraphSpace#320]
The UI has a limit for the graph title when uploading a graph while the REST API seemingly does not. The database, UI and REST API should all have the same limit.

**Comment by tmmurali** [comment]
@sanket0211, thank you for your interest. Can you work on this issue?

**Comment by sanket0211** [comment]
Respected prof. @tmmurali, 
I am working on this issue. Will just increasing the allowed input size to 254 in the HTML page of upload graph work? I have checked the database schema. There is no limit defined for the column. The database stores Graph name in the form of a string whose max_length is not defined so I assume that it can store at least 254 characters. 
Thank you.

**Comment by tmmurali** [comment]
@adbharadwaj What do you think?

**Comment by adbharadwaj** [comment]
Few things to clarify!

- The UI only allows users to set the name of the graph. [Title](http://manual.graphspace.org/en/latest/GraphSpace_Network_Model.html#graph-data-attributes-attributes-treated-specially-by-graphspace) is a different field.
- The graph table has no `title` column. The title is stored in [graph data attributes](http://manual.graphspace.org/en/latest/GraphSpace_Network_Model.html#graph-data-attributes-attributes-treated-specially-by-graphspace) and have no length restrictions. 
- The graph table has `name` column of type text which can store unlimited length strings.However we would like to restrict users from entering very long names. So we can decide a accepted length and set that as a restriction via UI. I would say 256 is a reasonable name length.

@sanket0211 I believe you are trying to increase the input size of the name of the graph on the upload page. I would suggest 256 as the input length.

**Comment by tmmurali** [comment]
If we enforce a restriction on the length of an attribute, we need to
document this restriction. Where do we document it? Do we document why we
have such a restriction?

On Mon, Feb 12, 2018 at 12:19 PM, Aditya Bharadwaj <notifications@github.com

**Comment by adbharadwaj** [comment]
The reason is to avoid very long graph names which can hamper the search page look and feel.

We can document it above the input field in HTML code.

**Comment by tmmurali** [comment]
Yes, there but also in the main documentation of GraphSpace, both in docs
of the user interface and the docs for the REST API/GraphSpace Python.

On Mon, Feb 12, 2018 at 12:27 PM, Aditya Bharadwaj <notifications@github.com

**Comment by adbharadwaj** [comment]
As of now, there is no limit check on API code. @sanket0211 Can you make that change and update docs for the REST API>

**Comment by sanket0211** [comment]
Prof. @tmmurali, @adbharadwaj Sir, 
sorry for late reply. I was not keeping well yesterday. 

Prof. @tmmurali, thank you for your reply. @adbharadwaj Sir, yes I 'll make that change and also update the corresponding docs too. 

Thank you.

**Comment by sanket0211** [comment]
@adbharadwaj  Sir, 
I have added a patch. Can you please review and let me know if any changes are required?
Thank you.
